parm/fv3lam_rrfs.xml

```diff
@@ -141,82 +141,90 @@
        <scale>3.0</scale>
        </param>
 
+       <param>
+       <shortname>CICE_ON_HYBRID_LVL</shortname>
+       <pname>CICE</pname>
+       <level>1.</level>
+       <scale>3.0</scale>
+       </param>
+
        <param>
        <shortname>HGT_ON_ISOBARIC_SFC</shortname>
        <pname>HGT</pname>
        <level>200. 500. 700. 1000. 2000. 3000. 5000. 7000. 7500. 10000. 12500. 15000. 17500. 20000. 22500. 25000. 27500. 30000. 32500. 35000. 37500. 40000. 42500. 45000. 
-47500. 50000. 52500. 55000. 57500. 60000. 62500. 65000. 67500. 70000. 72500. 75000. 77500. 80000. 82500. 85000. 87500. 90000. 92500. 95000. 97500. 100000.</level>
+47500. 50000. 52500. 55000. 57500. 60000. 62500. 65000. 67500. 70000. 72500. 75000. 77500. 80000. 82500. 85000. 87500. 90000. 92500. 95000. 97500. 100000. 101320.</level>
```

Comment:
@AndrewBenjamin-NOAA are you removing 101320 level as you indicated at 5/16/2023 RRFS bi-weekly?

Reply:
Eric James was on vacation has been on leave and is now at a conference, I am waiting for him to return to give us the answer before removing it. But I would assume ultimately it will be removed, just doing do diligence.

Reply:
@EricJames-NOAA no problem, this was just brought up last week at our EMC bi-weekly. As you mentioned, 1013.2 level is specific to the hrrr.

At the bi-weekly, it was suggested I touch base with you to see if there is any reason why we need to keep this level? I am not sure why it didn't show up in my regression test, as it was just interpolating to a new pressure level, but before troubleshooting it was suggested to see if we can just remove this level and add it to the PNS.

Reply:
@AndrewBenjamin-NOAA I talked with Curtis, I think it's OK to remove the 1013.2 level.

Reply:
@

> @AndrewBenjamin-NOAA I talked with Curtis, I think it's OK to remove the 1013.2 level.

@EricJames-NOAA thank you !
@AndrewBenjamin-NOAA, I added removal of 1013.2 pressure level to the google doc I prepared for PNS:

[https://docs.google.com/document/d/1MqCARdxKEtAEdL0xTGrlaj_Ygvi6h7gEFCqd1oyNwsI/edit](url)

Reply:
@AndrewBenjamin-NOAA please also remove this level from your PR

Reply:
In the process of removing it, taking another look at FRACC first, including previous email chain for further discussion, it seems there is some confusion.